playbooks/templates/tripleo-deploy.sh.j2

```diff
@@ -8,7 +8,9 @@ sudo openstack tripleo deploy \
     --local-ip={{ control_plane_ip }} \
     --control-virtual-ip {{ public_api }} \
     -e /usr/share/openstack-tripleo-heat-templates/environments/standalone/standalone-tripleo.yaml \
-    -e /usr/share/openstack-tripleo-heat-templates/environments/services/octavia.yaml \
+{% for service in enabled_services %}
+    -e {{ service }} \
```

Comment:
If we don't have a need to enable services defined outside of `/usr/share/openstack-tripleo-heat-templates/environments/services/`, can I suggest we do like this:
```suggestion
    -e /usr/share/openstack-tripleo-heat-templates/environments/services/{{ service }}.yaml \
```

This would make the `enabled_services` variable more natural to use.